doc/api/tls.markdown

```diff
@@ -203,7 +203,8 @@ automatically set as a listener for the [secureConnection][] event.  The
 
   - `dhparam`: DH parameter file to use for DHE key agreement. Use
     `openssl dhparam` command to create it. If the file is invalid to
-    load, it is silently discarded.
+    load, it is silently discarded. Its key length should be greater
+    than or equal to 768 bits, otherwise it throws an error.
```

Comment:
"its key length" -> "it throws an error", with "it" referring to two different things here. Maybe just "otherwise an error will be thrown" as the easy way out.